src/server/game/Entities/Player/Player.cpp

```diff
@@ -26960,3 +26955,8 @@ std::string Player::GetDebugInfo() const
     sstr << Unit::GetDebugInfo();
     return sstr.str();
 }
+
+GameClient* Player::GetGameClient() const
+{
+    return GetSession()->GetGameClient();
```

Comment:
should there be a null check for GetSession() ? are there cases where a Player entity still exists while the session doesn't anymore ? I don't remember

Reply:
no